lib/ecmascript.ts

```diff
@@ -1200,7 +1213,7 @@ interface FieldPrepareOptions {
   emptySourceErrorMessage: string;
 }
 
-type AnyTemporalKey = Exclude<Keys<AnyTemporalLikeType>, symbol>;
+export type AnyTemporalKey = Exclude<Keys<AnyTemporalLikeType>, symbol>;
```

Comment:
If we don't use this exported type per comment above, then shouldn't export this. And if it is shared, then we may want to move it into `intnernaltypes.d.ts`.

Reply:
Done.